charts/rstudio-connect/NEWS.md

```diff
@@ -1,3 +1,7 @@
+## 0.7.4
```

Comment:
This (and 0.7.3) should go after `# Changelog`, no?
and 0.7.3 should use double-hash for h2 heading.

Reply:
Good catch. We missed [this change](https://github.com/rstudio/helm/commit/8a327f31196c235b08f08f57996d56ae9434acff) which added the changelog header. I'll update the Connect release automation to account for this when we write the news entry.

Reply:
@colearendt I've updated our automation in connect so that this wont happen during our next release but it does depend on this PR getting merged because `# Changelog` needs to be at the beginning of the NEWS file.